docs/how-to/h-create-and-list-backups.md

```diff
@@ -0,0 +1,14 @@
+Creating and listing backups requires that you:
+* [Have a cluster with at least three nodes deployed](https://discourse.charmhub.io/t/charmed-mysql-tutorial-managing-units/TODO)
```

Comment:
This is not a requirement for creating or listing a backup. We can create a backup or list backups if we have 1 node deployed

Reply:
Refactored.